test/com/esotericsoftware/kryo/serializers/CompatibleFieldSerializerTest.java

```diff
@@ -264,7 +266,7 @@ private void testChangePrimitiveAndWrapperFieldTypes (int length, boolean chunke
 		otherKryo.register(ClassWithWrapperAndPrimitive.class, otherSerializer);
 
 		ClassWithWrapperAndPrimitive o = (ClassWithWrapperAndPrimitive) otherKryo.readClassAndObject(input);
-		assertEquals(1L, o.value1, 0);
+		assertEquals(1L, o.value1, EPS);
```

Comment:
Please remove the delta argument as you suggested.

Reply:
This is now done in my second commit, I used the assertEquals(long, long) form and the longValue() method to get a long from a Long.